lib/options.json

```diff
@@ -1031,7 +1031,7 @@
           "$ref": "#/definitions/WebSocketServerType"
         }
       ],
-      "description": "Deprecated: please use 'webSocketServer.type'/'--web-socket-server-type' option."
+      "description": "Deprecated: please use '--web-socket-server-type' option."
```

Comment:
not sure if displaying the flag makes sense when the user is using the feature from config

Reply:
Ideally, we should have a `cliDescription` property which we should respect if available. https://github.com/webpack/webpack/blob/c854e30df7f879d9989a7ab7aa83d1f0d1fa6862/lib/cli.js#L97